### PR TITLE
Terraform updates for Malta STG

### PR DESCRIPTION
**Pull request by janorivera**
Primary reviewer:
@GPaoloni  or @stephenhand 
FYI @Humairaa127 
Terraform configuration for MT Staging.

All terraform changes have been applied already and tested.

### Checklist
- [x] Enabling Facebook channel
- [x] Creating language chatbots
- [x] Creating custom studio flows
- [x] Terraform apply run

### Related Issues
Fixes #....

### CODE REVIEW

File: twilio-iac/mt-kellimni-staging/main.tf [hunk 1]
```diff
@@ -21,11 +21,13 @@ locals {
   environment = "Staging"
   short_environment = "STG"
   definition_version = "mt-v1"
+  helpline_language = "en-MT"
   permission_config = "mt"
   multi_office = false
   enable_post_survey = false
   target_task_name = "greeting"
-  twilio_numbers = [""]
+  task_language = "en-MT"
```
Comment (GPaoloni) [inline twilio-iac/mt-kellimni-staging/main.tf:29]: Shouldn't this one be set per-task  ?

File: twilio-iac/terraform-modules/channels/twilio-channel/channel-attributes-mt/sms-attributes.tftpl [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+  "name": "{{trigger.message.ChannelAttributes.from}}",
+  "channelType": "{{trigger.message.ChannelAttributes.channel_type}}",
+  "channelSid": "{{trigger.message.ChannelSid}}",
+  "twilioNumber": "{{trigger.message.ChannelAttributes.twilioNumber}}",
+  "ignoreAgent": "",
+  "language":"${task_language}",
```
Comment (GPaoloni) [inline twilio-iac/terraform-modules/channels/twilio-channel/channel-attributes-mt/sms-attributes.tftpl:7]: This one is not part of the bot survey for SMS?

File: twilio-iac/terraform-modules/channels/twilio-channel/variables.tf [hunk 3]
```diff
@@ -18,6 +18,7 @@ variable "custom_channel_attributes" {
 
 variable "pre_survey_bot_sid" {
   description = "Internal Twilio resource SID provided by another module"
+  default = ""
```
Comment (GPaoloni) [inline twilio-iac/terraform-modules/channels/twilio-channel/variables.tf:21]: Why this one needs a default?

**Review by GPaoloni — APPROVED** [review]
LGM! Just leaving Qs, no blockers tho.

File: twilio-iac/terraform-modules/channels/twilio-channel/variables.tf (hunk 3 above)
Comment (janorivera) [reply twilio-iac/terraform-modules/channels/twilio-channel/variables.tf:21]: the module expects that variable to create the default flow, even though it is not being used in this case.

File: twilio-iac/terraform-modules/channels/twilio-channel/channel-attributes-mt/sms-attributes.tftpl (hunk 2 above)
Comment (janorivera) [reply twilio-iac/terraform-modules/channels/twilio-channel/channel-attributes-mt/sms-attributes.tftpl:7]: not sure if I understand the question , but we are not implementing SMS anytime soon.

File: twilio-iac/mt-kellimni-staging/main.tf (hunk 1 above)
Comment (janorivera) [reply twilio-iac/mt-kellimni-staging/main.tf]: not needed actually, so I've removed it.

**Pull request merged**